whois-commons/src/main/java/net/ripe/db/whois/common/dao/jdbc/JdbcVersionDao.java

```diff
@@ -144,4 +175,15 @@ public List<VersionInfo> getVersionsForTimestamp(final ObjectType type, final St
                 "WHERE temp.timestamp=:cutoffTime ", parameters, new VersionInfoRowMapper()));
         return versionInfos;
     }
+
+    public List<SerialRpslObjectTuple> findSerialsInLastSince(final int serialId) {
+        final String sql = String.format(sqlSerialAndRpslTemplate, "last");
+        return jdbcTemplate.query(sql, serialAndRpslObjectMapper, serialId);
+    }
```

Comment:
Can we create a method which returns result of jdbcTemplate.query(sql, serialAndRpslObjectMapper, serialId); and pass on the "last" or "history" ?
This way we can get rid of unnecessary sql query constant at the top

Reply:
Also this way we can use the serialAndRpslObjectMapper directly

Reply:
done